frontend/src/core/less/fonts.less

```diff
@@ -17,27 +18,10 @@ body {
 h1 {
   font-size:15px;
   font-family: @font-title;
-  font-weight:300;
 }
 
-h2 {
-  font-weight:300;
-}
-
-h3 {
-  font-weight:300;
-}
-
-h4 {
-  font-weight:200;
-}
-
-h5 {
-  font-weight:200;
-}
-
-h6 {
-  font-weight:200;
+h1, h2, h3, h4, h5, h6 {
```

Comment:
> * Revert header tags to bold weighting, removing the need for multiple overrides

We have been overriding in several places just to get back to the (browser) default of bold weights for headings, making things a little convoluted. Also, the lighter weights have never been rendered from the offset, rather falling back to `normal`/`400`.